utils/chainId.ts

```diff
@@ -1,8 +1,11 @@
 import EthereumLogo from "../assets/ethereum.svg";
 import PolygonLogo from "../assets/polygon.svg";
+import ArbitrumLogo from "../public/icons/arbitrum.svg";
+import OptimismLogo from "../public/icons/optimism.svg";
+import BobaLogo from "../public/icons/boba.svg";
 
 export type ValidChainId = 1 | 42 | 137 | 1337;
-export type ChainId = 1 | 42 | 1337 | 3 | 4 | 137;
+export type ChainId = 1 | 42 | 1337 | 3 | 4 | 137 | 10 | 42161 | 288;
```

Comment:
nothing to change here, I would say we should prob use an enum in the future for more type safety and legibility